extensions/amp-story/0.1/page-advancement.js

```diff
@@ -276,8 +276,12 @@ class ManualAdvancement extends AdvancementConfig {
 
     // TODO(newmuis): This will need to be flipped for RTL.
     const elRect = this.element_./*OK*/getBoundingClientRect();
-    const offsetLeft = elRect.x;
+
+    // Using `left` as a fallback since Safari returns a ClientRect in some
+    // cases.
+    const offsetLeft = elRect.x || elRect.left;
```

Comment:
Perhaps check `== null` instead to avoid triggering fallback on `x == 0`.

Reply:
Thanks for catching that!